### PR TITLE
Strftime on expressions input string

### PR DESCRIPTION
**Pull request by petergrlica**
Receive string as an input parameter to the strftime expression function also.

### CODE REVIEW

File: pkg/expr/func_time.go [hunk 1]
```diff
@@ -173,6 +173,14 @@ func prepMod(base interface{}, mod interface{}) (*time.Time, int, error) {
 		t = &auxt
 	case *time.Time:
 		t = auxt
+	case string:
+		tt, err := cast.ToTimeE(auxt)
```
Comment (tjerman) [inline pkg/expr/func_time.go]: Should we use the same functions we do for record values and string-date conversion?

Comment (petergrlica) [reply pkg/expr/func_time.go]: These functions, we use all over the `expr` package, so I wanted to be consistent.
Can you provide an example of the conversions you mention?

Comment (tjerman) [reply pkg/expr/func_time.go]: https://github.com/cortezaproject/corteza-server/blob/2022.9.x-crs/compose/service/values/sanitizer.go#L193

Comment (petergrlica) [reply pkg/expr/func_time.go]: @tjerman I would like to leave the `compose/service` methods out of `pkg` area so as not to pollute (all that we include in this package is external libs / `pkg/*` as I think should be the case)

**Pull request closed without merging**